<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainscreen</class>
 <widget class="QDialog" name="mainscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Room</string>
  </property>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>75</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tb_chat_output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_chat_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
